public/documents/fari-games/charge-srd.md

```diff
@@ -7,19 +7,19 @@ highlightFont: Oswald
 
 # Introductions | Charge SRD
 
-This is a condensed version of the rules of Charge RPG that you can use to make your own **Powered by Charge** games.
+The Charge SRD is a condensed version of the rules of Charge RPG that you can use to make your own **Powered by Charge** games.
 
-This document follows a certain format that you can use as a base framework for your game.
```

Comment:
Why removing "This document"?

Reply:
I thought "it" would be more concise, but do you think it was better the way it was before?

Reply:
"It" would be perfect, if "This document" appeared before. With the new version of the previous line "it" refers to "The Charge SRD". It still makes sense, it's just a small shift as Charge SRD can be tought as a little more than a document, despite its name.
So both "it" and "this document" are ok, I just think the latter is better as it fits more with "follows a certain format".

Reply:
I'll leave it to "this document" !